change/@fluentui-react-components-6b800570-deea-41a1-a565-0987a0589347.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
```suggestion
  "type": "none",
```

I don't see any changes in production code:

![image](https://user-images.githubusercontent.com/14183168/157420276-d59dcf05-b695-46f7-9cf4-9aaa2a43cb86.png)

Should it be just `none`?